src/main/flight/interpolated_setpoint.c

```diff
@@ -58,9 +69,42 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float rxInterval = currentRxRefreshRate * 1e-6f;
         const float rxRate = 1.0f / rxInterval;
 
-        const float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
-        const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
+        float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
+        float setpointAcceleration = setpointSpeed - prevSetpointSpeed[axis];
+        const float holdSteps = 2.0f;
+
+        // Glitch reduction code for identical packets
+        if (setpointSpeed == 0 && fabsf(rawSetpoint) < 0.98f * ffMaxRate[axis]) {
+            // identical packets, not at full deflection
+            if (holdCount[axis] < holdSteps && fabsf(rawSetpoint) > 2.0f && !bigStep[axis]) {
+                // holding the entire previous speed is best for missed packets, but bad for early packets
+                setpointSpeed = prevSetpointSpeed[axis] + prevAcceleration[axis];
+                setpointAcceleration = prevAcceleration[axis];
+                holdCount[axis] += 1.0f;
+            // hit timeout, or at centre and not moving, or prev big step, lock speed and acceleration to zero
+            } else {
+                holdCount[axis] -= 1.0f;
+                setpointSpeed = 0.0f;
+                setpointAcceleration = 0.0f;
+            }
+        } else {
+            // we're moving!
+            if (holdCount[axis] == 2.0f) {
```

Comment:
Currently this makes sense, but what happens if you set holdSteps=3?  And you hold 2 frames and then get here with holdCount[axis]=3?

Maybe something like:

```
if (holdCount[axis] > 1) {
    setpointSpeed /= holdCount[axis];
    setpointAcceleration /= holdCount[axis];
}

So then if we hold for 2 frames, holdCount[axis] is then 3, and we would expect next 'good' package to be 3 times what it should be so we device by 3.
```

Reply:
I originally did code it to potentially handle holds for more than one step, but to keep it simple enough for now, it only manages one single missed packet.  That was complicated enough.  The problem what to do after re-starting after more than one identical packet is knowing whether the train of identical packets was real or not.  If the user held the sticks still, or wasn't moving much, or had ADC on, then we get runs of identical packets and not necessarily a big step up.  If we attenuate that step up after say 10 identical packets, and actually it is a 'normal' step up like with ADC on, that would be bad.  
I was hoping to sort this code out as is, then extend it possibly for longer gaps.

Reply:
I won't do this for now, but it is something we could consider later, it is a bit tricky to test.